app/src/main/res/values/dimens.xml

```diff
@@ -6,7 +6,10 @@
     -->
     <dimen name="box_inset_layout_padding">0dp</dimen>
     <dimen name="floatingActionButton01" tools:override="true">36dp</dimen>
-
+    <dimen name="sm">5dp</dimen>
+    <dimen name="md">8dp</dimen>
+    <dimen name="lg">120dp</dimen>
+    <dimen name="xlg">136dp</dimen>
```

Comment:
Added constants for space sizing to have consistency instead of had coding.

Reply:
Good idea for parameterization.